apps/public-docsite-v9/src/shims/ThemeShim/v9ThemeShim.ts

```diff
@@ -108,7 +108,9 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralBackgroundInvertedDisabled: whiteAlpha[10],
     colorNeutralStencil1: palette.neutralLight,
     colorNeutralStencil2: palette.neutralLighterAlt,
-    colorBackgroundOverlay: blackAlpha[10],
+    colorNeutralStencil1Alpha: inverted ? whiteAlpha[10] : blackAlpha[10],
+    colorNeutralStencil2Alpha: inverted ? whiteAlpha[10] : blackAlpha[5],
```

Comment:
According to the PR description

```suggestion
    colorNeutralStencil2Alpha: inverted ? whiteAlpha[5] : blackAlpha[5],
```